libs/features/mas/src/price/index.js

```diff
@@ -18,6 +18,9 @@ const priceStrikethrough = createPriceTemplate({
 const priceAnnual = createPriceTemplate({
     displayAnnual: true,
 });
+const priceAlternative = createPriceTemplate({
+    isAlternativePrice: true,
```

Comment:
I wonder if we need to pass also 
```
displayOptical = false,
displayStrikethrough = false,
displayAnnual = false,
```

Reply:
We don't need since they are false in default. 
<img width="444" alt="image" src="https://github.com/user-attachments/assets/b6a408dd-3f86-4c3e-8e63-ed1b3143d3d4" />

Reply:
but since you pass `{isAlterntivePrice}` afaik every other parameter will be set to undefined.

Reply:
same as for templates above, no?
i guess it works fine

Reply:
I tested and they are still `false` as default value. So, I think it's good as it is.
<img width="547" alt="image" src="https://github.com/user-attachments/assets/8e2a879a-f61c-41b2-a5fe-4e8e05d0dc42" />
<img width="289" alt="image" src="https://github.com/user-attachments/assets/91c06b2f-4e78-453a-894d-934801b80d75" />